chat-plugins/poll.js

```diff
@@ -152,12 +151,21 @@ exports.commands = {
 			room.poll.end();
 			delete room.poll;
 		},
+		endhelp: ["/poll end - Ends the poll and displays the results in chat. Requires: % @ # & ~"],
 
 		'': function (target, room, user) {
 			if (!room.poll) return this.errorReply("There is no poll running in this room.");
 			if (!this.canBroadcast()) return;
+			room.update();
 
 			room.poll.display(user, this.broadcasting);
 		}
-	}
+	},
+	pollhelp: ["Poll plugin for Pokémon Showdown:",
+				"This plugin allows rooms to run their own polls. These polls are limited to one poll at a time per room, and display to the room when they're created, ended, or when someone joins the room.",
+				"The poll is displayed in a textbox, and is updated in real time. It's also possible to vote by clicking the buttons on the poll itself.",
+				"This plugin contains the following commands:",
+				"/poll new [name] | [options] - Starts a new poll. The amount of options can be between 1 and 8. Requires: % @ # & ~",
```

Comment:
Just tested it, it actually doesn't even let you create a poll with 1 option anyways, so yea I like Slayer's suggestion.